ios/FluentUI/Drawer/DrawerController.swift

```diff
@@ -408,6 +409,8 @@ open class DrawerController: UIViewController {
     private var useCustomBackgroundColor: Bool = false
     /// for iPad split mode, navigation bar has a different dark elevated color, and if it is a `.down` presentation style, match `Colors.NavigationBar.background` elevated color
     private var useNavigationBarBackgroundColor: Bool = false
+    
+    private var preferredMaximumExpansionHeight: CGFloat
```

Comment:
private let? (this is not changed anywhere, right?)

Reply:
updated it.